src/content/docs/en/reference/configuration-reference.mdx

```diff
@@ -492,13 +492,17 @@ Specifies the directory in the build output where Astro-generated assets (bundle
 
 <p>
 
-**Type:** `string`<br />
+**Type:** `(string | Record.<string, string>)`<br />
```

Comment:
```suggestion
**Type:** `string | Record.<string, string>`<br />
```

Reply:
I'm honestly not sure what's best to use here, but I don't see too many parentheses used, so I'll commit this and we'll figure out the tiny details when this is added to the other PR!  😄